### PR TITLE
Redesign portal modal

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1365
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1471
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1446

BEFORE
image

AFTER
image

### CODE REVIEW

**Comment by colinhowe** [comment]
Probably a question for design. When the modal gets too tall for the screen it becomes unusable. The attached screenshot is an exaggeration, yet, the problem stands.
image

File: src/components/templates/PartyMap/components/PortalModal/PortalModal.tsx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+          className="PortalModal__description"
+          expanded={false}
+          truncatedEndingComponent={"... "}
+        >
+          <RenderMarkdown text={portalDescription} />
+        </ShowMoreText>
+      )}
+
+      <div className="PortalModal__btn-wrapper">
+        {/* @debt extract this 'enter room' button/link concept into a reusable component */}
```
Comment (colinhowe) [inline src/components/templates/PartyMap/components/PortalModal/PortalModal.tsx:200]: 'enter room' -> 'enter portal'
Let's try and remove the word room if we can

**Review by colinhowe — APPROVED** [review]
Minor change to a comment. Otherwise, great to go. No further review needed.

File: src/components/templates/PartyMap/components/PortalModal/PortalModal.tsx (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/templates/PartyMap/components/PortalModal/PortalModal.tsx:200]: I may suggest `ROOM_TAXON.lower` it's clunky, but at least we have a reference to all user facing concepts that we can use to quickly find them or modify it.

Comment (sunny-viktoryia) [reply src/components/templates/PartyMap/components/PortalModal/PortalModal.tsx]: done here: https://github.com/sparkletown/sparkle/pull/2588/commits/ebbc2894098218867a7a3a8ebe54bae6b0732396

**Pull request merged**